### ISSUE TITLE
AddOn versus Integration

### ISSUE THREAD

**Issue by hklages** [junalmeida/homeassistant-torque#5]
Just a comment:
Is it really a Home Assistant "AddOn"? Typically Add Ons are Docker container. Your code looks more like an "integration". 

A question:
There is already an integration named Torque, are you aware of that? 

Next week I will receive a OBD2 device and give try yours and the other one

**Comment by junalmeida** [comment]
Yes it is an integration, the code is not yet released and I have some things to change before that, including the documentation.

**Comment by junalmeida** [comment]
And yes, I'm aware, however my idea is to make another one more complete. For eg this one will provide a `device_tracker` and make data more persistent in the case you use it with phones or do not use your car in a while. On my case, the "official" integration's entities disappear after a while not receiving data.

**Comment by junalmeida** [comment]
This is under alpha stage yet. Thanks for being an early tester :)

**Comment by hklages** [comment]
Understand.

**Comment by hklages** [comment]
Hi, just a short update, for your information.

I got my OBD2 adapter, installed Torque app on my smartphone, added the torque platform (it is a platform not an integration) at HA, use the nabu casa remote access with bearer. All selected PIDs are available as sensors. I created one additional template sensor and added the gps data as attributes - thats only to be able to use the map card with an entity. 

My use cases are: last parking position (with map card) and fuel refill planning.  That works fine for me.
[link](https://github.com/hklages/node-red-contrib-sonos-plus/wiki/miscellaneous#car---obd2)
